polygeist/include/mlir/Dialect/Polygeist/Transforms/Passes.h

```diff
@@ -50,7 +51,6 @@ std::unique_ptr<Pass> createOpenMPOptPass();
 std::unique_ptr<Pass> createParallelLowerPass();
 std::unique_ptr<Pass> createRaiseSCFToAffinePass();
 std::unique_ptr<Pass> createRemoveTrivialUsePass();
-std::unique_ptr<Pass> detectReductionPass();
 std::unique_ptr<Pass> replaceAffineCFGPass();
```

Comment:
do this one as well if you want the names to be consistent.

Reply:
done.